dadi/lib/handlers/image.js

```diff
@@ -382,11 +384,18 @@ ImageHandler.prototype.convert = function (stream, imageInfo) {
                 (width / height) !== (imageInfo.width / imageInfo.height) &&
                 !self.storageHandler.notFound
               ) {
+                console.log({
+                  left: crops.x1,
+                  top: crops.y1,
+                  width: crops.x2 - crops.x1,
+                  height: crops.y2 - crops.y1
+                })
+
                 sharpImage.extract({
                   left: crops.x1,
                   top: crops.y1,
-                  width: crops.x1 + crops.x2,
-                  height: crops.y1 + crops.y2
+                  width: crops.x2 - crops.x1,
```

Comment:
Oddly though, I've only just made that change. Perhaps you were testing a different resize option further up...

Regression tests... We'll need to work out a way to do that, Niffy isn't going to work due to the difference in sharpness/quality. We need something a little less _thorough_